code/modules/organs/internal/species/machine/internal_storage.dm

```diff
@@ -0,0 +1,42 @@
+/obj/item/organ/internal/machine/internal_storage
+	name = "internal storage system"
+	desc = "A simple internal casing used by G2 frames for internal storage."
+	organ_tag = BP_INTERNAL_STORAGE
+	parent_organ = BP_GROIN
+
+	action_button_name = "Access Internal Storage"
+
+	/// The actual, physical storage item.
+	var/obj/item/storage/internal_storage/storage
```

Comment:
If we add internal storage, there needs to be a accessible way for security to check if it is empty and possibly forcefully empty it.